clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -208,6 +209,7 @@ export class CopybookDownloadService {
   public async downloadCopybooks(
     documentUri: string,
     copybookNames: CopybookName[],
+    outputChannel?: vscode.OutputChannel,
```

Comment:
There is `outputChannel` member.
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/pull/2392/files#diff-c9b74f01a4b43df586e9ec106f05a4c816e4e617eb66a7ea0cff49b0f009f18dR178-R186